### PR TITLE
refactor(api, worker, application-generic, dal, shared): Rename "ControlVariables" to "ControlValues"

### PR DESCRIPTION
**Pull request by rifont**
### What changed? Why was the change needed?

* Rename "ControlVariables" to "ControlValues"
  * we previously had mixed usage of both "ControlVariables" and "ControlValues" which made consuming the control values repository and associated logic confusing.

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: libs/dal/src/repositories/control-values/controlValues.repository.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { SoftDeleteModel } from 'mongoose-delete';
-import { ControlVariablesLevelEnum } from '@novu/shared';
-import { ControlValuesModel, ControlVariables } from './controlVariables.schema';
-import { ControlValuesEntity } from './controlValuesEntity';
+import { ControlValuesLevelEnum } from '@novu/shared';
+import { ControlValuesModel, ControlValues } from './controlValues.schema';
+import { ControlValuesEntity } from './controlValues.entity';
```
Comment (djabarovgeorge) [inline libs/dal/src/repositories/control-values/controlValues.repository.ts]: 👏

Comment (djabarovgeorge) [reply libs/dal/src/repositories/control-values/controlValues.repository.ts]: can we please update this collection to use the current file conversion? this means changing the files to:
control-values.entity.ts
control-values.reposotiry.ts
control-values.schema.ts

**Review by djabarovgeorge — APPROVED** [review]
looks good to me, left one comment ;)

**Review by SokratisVidros — APPROVED** [review]
👏 👏 👏

File: libs/dal/src/repositories/control-values/controlValues.repository.ts (hunk 1 above)
Comment (rifont) [reply libs/dal/src/repositories/control-values/controlValues.repository.ts]: Good catch! I have updated.

**Pull request merged**